### PR TITLE
examples/step-5: Update indenting

### PR DESCRIPTION
**Pull request by tamiko**
In reference to #6774

### CODE REVIEW

**Comment by drwells** [comment]
The TODO refers to the lines (back in commit a594bc236)

```cpp
				 // What we will show here is how we
				 // can avoid calls to the
				 // shape_value, shape_grad, and
				 // quadrature_point functions of the
				 // FEValues object, and in particular
				 // optimize away most of the virtual
				 // function calls of the Function
				 // object. The way to do so will be
				 // explained in the following, while
				 // those parts of this function that
				 // are not changed with respect to
				 // the previous example are not
				 // commented on.
				 //
```

This comment survived into the present day in lines 161-165. We don't do anything unusual to avoid calling those `FEValues` methods in the following function, so could you just delete that whole comment to close the TODO?

File: examples/step-5/step-5.cc [hunk 1]
```diff
@@ -203,11 +197,14 @@ void Step5<dim>::assemble_system()
...
                 cell_matrix(i, j) +=
-                  (current_coefficient * fe_values.shape_grad(i, q_index) *
-                   fe_values.shape_grad(j, q_index) * fe_values.JxW(q_index));
-
-              cell_rhs(i) += (fe_values.shape_value(i, q_index) * 1.0 *
-                              fe_values.JxW(q_index));
+                  (current_coefficient *              // a(x_q)
+                   fe_values.shape_grad(i, q_index) * // grad phi_i(x_q)
+                   fe_values.shape_grad(j, q_index) * // grad phi_j(x_q)
+                   fe_values.JxW(q_index));           // dx
```
Comment (tamiko) [inline examples/step-5/step-5.cc]: @bangerth Actually, the comments increase the readibility quite a bit. I like the current form.

**Comment by tamiko** [comment]
@drwells *ping*

File: examples/step-5/step-5.cc (hunk 1 above)
Comment (bangerth) [reply examples/step-5/step-5.cc]: I agree. If you can make lemonade out of lemons, you shouldn't complain :-)

**Pull request merged**